dev/RadioButtons/RadioButtons.cpp

```diff
@@ -561,37 +581,37 @@ void RadioButtons::UpdateSelectedItem()
     }
 }
 
-winrt::ToggleButton RadioButtons::TryGetRadioButtonFromRepeaterElement(const winrt::UIElement& element)
+winrt::UIElement RadioButtons::TryGetRadioButtonAsUIElement(const winrt::UIElement& repaterElement)
 {
-    if (auto const elementAsGrid = element.try_as<winrt::Grid>())
+    if (auto const elementAsGrid = repaterElement.try_as<winrt::Grid>())
```

Comment:
I think Panel would suffice instead of grid.

Reply:
Done.